docs/products/kafka/howto/view-kafka-topic-catalog.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Manage Apache Kafka topics with topic catalog
+sidebar_label: Manage topic catalog
+limited: true
+---
+
+The Aiven for Apache Kafka® topic catalog offers a user-friendly interface to manage your Apache Kafka topics within your Aiven for Apache Kafka services.
+
+## Access topic catalog
+
+1. Log in to the [Aiven console](https://console.aiven.io/).
+1. Click **Tools**.
+1. Select **Apache Kafka topic catalog**.
+
+## Browse the topic catalog
+
+On the Apache Kafka topic catalog page, you can:
+
+- **View topics in a table**: This default view lists all topics in a table format, showing columns
+  for the topic name, service, project, and owner.
+- **Switch to card view**: Click **Tiles** to switch to the card view and browse topics displayed
+  as cards.
+- **Search for topics**: Enter the topic name in the search bar at the top
+  to find specific topics.
+
+## Request a new topic
+
+You can request the creation of a new topic from the Apache Kafka topic catalog page,
+which goes through an approval process to ensure proper governance.
+
+To request a new topic:
+
+1. On the Apache Kafka topic catalog page, click **Request new topic**.
+1. Select the project.
+1. Select the Aiven for Apache Kafka service.
+1. Select the group for topic ownership.
+1. Enter a unique topic name.
+1. Set the replication factor (maximum value: 3).
```

Comment:
(maximum value: 3) and (maximum value: 8) are dependent on the global topic settings, so those would be the defaults but organizations could have those set to much higher numbers.

Reply:
Do i think we can just remove those references here